### ISSUE TITLE
Feature image as background image and Item ratio

### ISSUE THREAD

**Issue by alirexa** [dudaster/eletheme#32]
Any hope or trick for the feature image to be set as item background?
Also do you have the plan for implementing something like elementor pro's portfolio "Item ratio"?
Any trick for doing it via CSS? I need the Items to have 1:1 ratio.

**Comment by dudaster** [comment]
Is in the works.

**Comment by alirexa** [comment]
any due date?

**Comment by dudaster** [comment]
Unfortunatly not. In my spare time. I'm a webdev/designer and you know how this time of the year is.

P.S. Have you tried using call to action widget inside a loop template?

**Comment by alirexa** [comment]
Yes it is hard to find the time for a non commercial project with another important commitment, hope you find the time to develop this and then this type of work becomes your main income source and work :)

and no I didn't try that before but now I did and it works 👍 but there is another problem, I need my archive Items to have 1:1 ratio and it doesn't have the option, I found out for a div to have equal width and height and being responsive, the combination of CSS width and padding-top may be the solution : 

`.theSquareDiv{
width:100%;
padding-top:100%
}`
example here: https://www.w3schools.com/howto/tryit.asp?filename=tryhow_css_aspect_ratio

but somehow I couldn't use this method with elementor's block. do you know how could it be done with call to action or maybe other elements?

**Comment by dudaster** [comment]
Try css on article tag

**Comment by alirexa** [comment]
I tried that but it didn't work, I asked the question in Elementor's facebook community too but still no right answer :/
